packages/react-components/src/organisms/TeamCreateOutputForm.tsx

```diff
@@ -107,6 +109,10 @@ const TeamCreateOutputForm: React.FC<TeamCreateOutputFormProps> = ({
           authors: authors.map(({ value }) => value),
           labs: labs.map(({ value }) => value),
           teams: teams.map(({ value }) => value),
+          accessInstructions:
+            String(accessInstructions).trim() !== ''
```

Comment:
Not sure what this is doing. We require a value but this means if you put in " " you can send empty values?